packages/father-build/package.json

```
@@ -39,17 +46,10 @@
     "less-plugin-npm-import": "2.1.0",
     "lodash": "4.17.19",
     "rimraf": "2.6.3",
-    "rollup": "1.27.8",
-    "rollup-plugin-babel": "4.3.3",
-    "rollup-plugin-commonjs": "10.0.0",
-    "rollup-plugin-inject": "3.0.1",
-    "rollup-plugin-json": "4.0.0",
-    "rollup-plugin-node-resolve": "5.0.1",
-    "rollup-plugin-postcss-umi": "2.0.3",
-    "rollup-plugin-replace": "2.2.0",
-    "rollup-plugin-terser": "5.1.3",
+    "rollup": "2.33.1",
```

前几天又发了个新版： https://github.com/rollup/rollup/releases/tag/v2.33.2

```suggestion
    "rollup": "2.33.2",
```